resolwe_bio/processes/import_data/expressions.yml

```diff
@@ -71,7 +71,8 @@
         re-save exp_type ${EXP_TYPE}
         re-save-file exp ${NAME}.tab.gz
         re-save-file rc ${NAME}.tab.gz
-        expression2storage.py "${NAME}.tab.gz"
+        expression2storage.py "${NAME}.tab.gz" --output json.txt
```

Comment:
Usually positional arguments are passed before the file:
```
expression2storage.py --output json.txt "${NAME}.tab.gz"
```

Have a look at `expression2storage.py --help`.